easybuild/easyconfigs/a/Anaconda2/Anaconda2-4.0.0.eb

```diff
@@ -0,0 +1,21 @@
+# author: Jillian Rowe <jillian.e.rowe@gmail.com>
+easyblock = 'EB_Anaconda'
+
+name = 'Anaconda2'
+version = '4.0.0'
+
+homepage = 'https://www.continuum.io/anaconda-overview'
+description = """Built to complement the rich, open source Python community,
+the Anaconda platform provides an enterprise-ready data analytics platform 
+that empowers companies to adopt a modern open data science analytics architecture.
+"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://repo.continuum.io/archive/']
+sources = ['%(name)s-%(version)s-Linux-x86_64.sh']
+#checksums = ['31ed3ef07435d7068e1e03be49381b13' ]
```

Comment:
@jerowe why is this skipped? If the checksum is not right, we should correct it instead?